lib/rules/jsx-indent.js

```diff
@@ -290,6 +295,51 @@ module.exports = {
       }
     }
 
+    /**
+     * Instead String.prototype.mathAll
+     * @param {String} str
+     * @param {RegExp} regex
+     * @returns {Array}
+     */
+    function matchAll(str, regex) {
```

Comment:
there's no need to hardcode this when https://npmjs.com/string.prototype.matchall exists; please use that instead.